### PR TITLE
fixed logic for stepper

### PR DESCRIPTION
**Pull request by alex-cory**

### CODE REVIEW

File: src/helpers/cardInfoUtil.js [hunk 1]
```diff
@@ -52,12 +52,12 @@ class CardInfoUtil {
       },
       {
         title: BETTING,
-        description: `${getLocalDateTimeString(cOracle.startTime)} 
+        description: `${getLocalDateTimeString(cOracle.startTime)}
```
Comment (dwalintukan) [inline src/helpers/cardInfoUtil.js]: does this next line work? I've had trouble where putting it on the next line didn't show the space that is needed.

File: src/scenes/Event/scenes/oracle.js [hunk 2]
```diff
@@ -357,7 +356,7 @@ class OraclePage extends React.Component {
             message: intl.formatMessage(messages.eventUnconfirmed),
           },
         };
-      } else if (token === Token.Qtum && status === OracleStatus.Voting) {
```
Comment (dwalintukan) [inline src/scenes/Event/scenes/oracle.js]: revert this change. we need `Token` to differentiate between a Centralized Oracle vs Decentralized Oracle. 

`status === voting` can be both for COracle and DOracle. (`Voting` is a synonym for `Betting`. Frank just decided to reuse the status.)

File: src/scenes/Event/scenes/oracle.js [hunk 3]
```diff
@@ -371,7 +370,7 @@ class OraclePage extends React.Component {
             btnText: <FormattedMessage id="str.bet" defaultMessage="Bet" />,
           },
         };
-      } else if (token === Token.Qtum && (status === OracleStatus.WaitResult || status === OracleStatus.OpenResultSet)) {
```
Comment (dwalintukan) [inline src/scenes/Event/scenes/oracle.js]: revert, same comment as above. the status is reused for both COracle and DOracle

File: src/scenes/Event/scenes/oracle.js [hunk 4]
```diff
@@ -389,7 +388,7 @@ class OraclePage extends React.Component {
             message: intl.formatMessage(messages.setResultExplanation),
           },
         };
-      } else if (token === Token.Bot && status === OracleStatus.Voting) {
```
Comment (dwalintukan) [inline src/scenes/Event/scenes/oracle.js]: revert

File: src/scenes/Event/scenes/oracle.js [hunk 5]
```diff
@@ -407,7 +406,7 @@ class OraclePage extends React.Component {
             message: intl.formatMessage(messages.voteExplanation),
           },
         };
-      } else if (token === Token.Bot && status === OracleStatus.WaitResult) {
```
Comment (dwalintukan) [inline src/scenes/Event/scenes/oracle.js]: revert

File: src/scenes/Event/scenes/oracle.js (hunk 2 above)
Comment (dwalintukan) [reply src/scenes/Event/scenes/oracle.js]: it looks like this entire block is wrapped in `if (oracle && oracle.token === Token.Qtum)` which is incorrect. 

`else if (token === Token.Bot && status === OracleStatus.Voting)` and `else if (token === Token.Bot && status === OracleStatus.WaitResult)` will never be reached (the last two cases).

File: src/helpers/cardInfoUtil.js (hunk 1 above)
Comment (alex-cory) [reply src/helpers/cardInfoUtil.js]: I didn't actually change this. I think it was my editor just removing spaces at the end of lines.

File: src/scenes/Event/scenes/oracle.js (hunk 2 above)
Comment (alex-cory) [reply src/scenes/Event/scenes/oracle.js]: sorry about that. My bad 😵
